infra/cmake/packages/FlatBuffers-2.0/FlatBuffersConfig.cmake

```diff
@@ -22,6 +22,9 @@ function(_FlatBuffers_build)
   if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 8.0)
     set(ADDITIONAL_CXX_FLAGS "-Wno-error=class-memaccess")
   endif()
+  if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 12.0)
+    set(ADDITIONAL_CXX_FLAGS "-Wno-error=stringop-overflow")
```

Comment:
maybe ?
```suggestion
    set(ADDITIONAL_CXX_FLAGS ${ADDITIONAL_CXX_FLAGS} "-Wno-error=stringop-overflow")
```

Reply:
I don't know the reason, but gcc>12 does not requires `-Wno-error=class-memaccess`.